dotCMS/src/main/java/com/dotcms/rest/api/v1/categories/CategoriesResource.java

```diff
@@ -0,0 +1,78 @@
+package com.dotcms.rest.api.v1.categories;
+
+import com.dotcms.contenttype.business.ContentTypeAPI;
+import com.dotcms.contenttype.exception.NotFoundInDbException;
+import com.dotcms.contenttype.model.type.ContentType;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotcms.contenttype.model.type.ContentType'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)